git_pile/git_pile.py

```diff
@@ -333,63 +332,6 @@ class Break(Exception):
         git(f"worktree remove {d}")
 
 
-def cmd_init(args, config):
```

Comment:
I think this commit and the previous one should rather be squashed. I don't like having the intermediate state with a lot of dead code that is then referenced. Even if I understand how this was generated by automation, I think it's better having the explicit diff with code move than having to review that the automation got everything right.

Reply:
Okay. I'll squash it.

FYI, my rationale:
My original intention was to make the separate commits to help review but mark them to be squashed when applying. After realizing that (i) I could make the two separate commits without breaking git-pile and (ii) it would not be a pure code move, as there was one command (`AmCmd`) that would need a change in the middle of the function, I decided to keep them separate (thinking about ourselves from the future reviewing the history).

Reply:
> (...) as there was one command (AmCmd) that would need a change in the middle of the function (...)

Just found another one, GenbranchCmd.

Reply:
> When it's possible, it's better to do the needed change in the middle of the function and then do the code move. Or even the other way around if it doesn't break.

That's the problem: I think would break either way, since the change in the middle is changing calls to old functions to use the new ones. That's why I did a "code copy" first and then a next commit fixing specific points of the newly copied code and removing the old one. That allows two non-breaking commits.

Reply:
I'll send an updated version of this PR soon. I'll send it with the commits squashed. Let me know if you prefer them separate (to aid review) and squash them prior to merge.

Reply:
Done and force-pushed.

Reply:
No, I was looking at the wrong place